pom.xml

```diff
@@ -50,6 +50,7 @@
     <junit.version>4.12</junit.version>
     <assertj.version>3.13.2</assertj.version>
     <jackson.version>2.9.9</jackson.version>
+    <jackson-databind.version>2.9.9.3</jackson-databind.version>
```

Comment:
I would prefer to inline this as it is a special-case.

Reply:
Jackson also has a BOM if we'd prefer using that instead. That's how I addressed this issue at work

Reply:
yeah I just noticed this in armeria also. I think we should use the bom instead, and mention we are using jackson bom for CVE. probably the original reason we overrode was CVE :P